sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -80,15 +83,20 @@ export interface ListRequestOptions {
 }
 
 /**
- * Represents result of create, get, update and delete operations on queue.
+ * Type representing just the server response `_response`
  */
-export type QueueResponse = QueueDetails & {
+export type EmptyResponse = {
```

Comment:
The response i.e the `_response` is not really empty is it?
Also, this type is being used as a base in other types.

I would suggest using `RestResponse` or something like that rather than "Empty" response.

@bterlson thoughts?

Reply:
Agree that `EmptyResponse` is a confusing/misleading as a name. I like `RestResponse`.

That said, given we don't know of other members than `_response` that would go on the base response type, how about just copying the `_response` type and getting rid of `EmptyResponse` entirely? /cc @ramya-rao-a

Reply:
I agree with @bterlson and prefer not having introducing this response type altogether. If it **has** to exist, then I would definitely want a better name than `Empty`Response

Reply:
It seems beneficial that code/docs duplication is minimized and reads better, imo.
Ok to not add if that's what we are more inclined on - reverted to duplicate `_response` field instead

Reply:
Duplicating the `_response` field in each Response type might feel like a code/doc duplication for us library authors, but trying to solve this by introducing a type adds unnecessary cognitive overload to the user who might be trying to figure out what the response type for say the CreateQueue operation looks like.

So, yes, I would very much suggest to not introduce the `EmptyResponse` type